mapie/classification.py

```diff
@@ -474,11 +494,25 @@ def _add_random_tie_breaking(
                 axis=1
             ), axis=1
         )
-        # compute V parameter from Romano+(2020)
-        vs = (
-            (y_proba_last_cumsumed - threshold.reshape(1, -1)) /
-            y_pred_proba_last[:, 0, :]
-        )
+
+        if self.method == "cumulated_score":
+            # compute V parameter from Romano+(2020)
```

Comment:
Specify what is V?

Reply:
That's why we put the reference, to define what is V in a mathematical way